src/model.d.ts

```diff
@@ -2093,6 +2093,11 @@ export abstract class Model<TModelAttributes extends {} = any, TCreationAttribut
    */
   _attributes: TModelAttributes; // TODO [>6]: make this a non-exported symbol (same as the one in hooks.d.ts)
 
+  /**
+   * Object that contains undelying model data
```

Comment:
```suggestion
   * Object that contains underlying model data
```

Reply:
@WikiRik thank you for taking a look.  i have made the suggested change and have pushed the comment change and the adding of the test.